pkg/backup/item_backupper_test.go

```diff
@@ -99,7 +109,12 @@ func TestBackupItemSkips(t *testing.T) {
 				backedUpItems: test.backedUpItems,
 			}
 
-			u := unstructuredOrDie(fmt.Sprintf(`{"apiVersion":"v1","kind":"Pod","metadata":{"namespace":"%s","name":"%s"}}`, test.namespace, test.name))
+			var deletionTimestamp string
```

Comment:
Maybe we should use an actual `Pod` and then the unstructured converter so we're not conditionally modifying json...